app/dashboard/models.py

```diff
@@ -4668,6 +4669,12 @@ class HackathonProject(SuperModel):
         choices=PROJECT_STATUS,
         blank=True
     )
+    message = models.CharField(
+        max_length=150,
```

Comment:
I'm wondering if this limit is too short, what do you think?

Reply:
i think is ok for a short message, originally the tweets have a length of 140 characters and worked well. One example could be:

`Hi! I'm python developer and I'm looking for a fronted partner to hack on the ETC bounty "Create A DIY Hardware Wallet Using Signatory"`

150 character limit will help to synthesize messages, and having a character count helper will help a lot the user. What do you think @PixelantDesign?